src/resource.js

```diff
@@ -1,6 +1,58 @@
 'use strict'
 
-export default () => {
+import url from 'url'
+import jts from 'jsontableschema'
+import _ from 'lodash'
 
-}
+export default class Resource {
 
+  constructor(descriptor) {
+    this._descriptor = descriptor
+    this.REMOTE_PROTOCOLS = ['http:', 'https:', 'ftp:', 'ftps:']
```

Comment:
But than we will introduce dependency (using `jts`). So I'd like to do so for such a simple constant. 

It's internal interface so it's pretty your call where to have it (agree it could be like `config.js` module or something). Here also OK. Just thought.